packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -106,9 +110,9 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
   }
 
   private _renderMultiLine(): React.ReactElement<React.HTMLAttributes<HTMLAreaElement>> {
-    const { theme } = this.props;
+    const theme = this._globalTheme;
```

Comment:
Can we not just call getTheme() here instead of keeping track of _globalTheme as a private class level variable?

Reply:
@Vitalius1 this is a good point, since we need it to pick up theme change events, so it needs to `getTheme()` every render. (Or you manually register to theme change events.)

Reply:
@phkuo isn't `getTheme` grabbing the default Fabric theme that as far as I know will always be the same. Why would we generate same exact theme on every render? Wasn't the point of this PR is have MessageBar ignore any contextual theme changes and always use the Fabric defaults?

Reply:
When you call `loadTheme` then the theme returned by `getTheme` will change. That's a global change, not a contextual change, and messageBar should still respond to global changes.

Reply:
Isn't in most cases calling `loadTheme` is to set up the contextual theme across the entire app? Won't we then end up having the same issue when we would switch to a dark mode theme across the entire app by calling `loadTheme`?

Reply:
The theme across the entire app is the global theme, it's not really considered a contextual theme.
The message bar already works for dark themes.
What we want to do here is make the message bar work in the global theme, so that the theme creator does not need to worry about supporting the message bar in autogenerated variant themes.